evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -26,12 +26,16 @@ def _make_rows(shap_values, normalized_values, pipeline_features, top_k, include
           list(str)
     """
     tuples = [(value[0], feature_name) for feature_name, value in normalized_values.items()]
-    tuples = sorted(tuples)
 
-    if len(tuples) <= 2 * top_k:
-        features_to_display = reversed(tuples)
-    else:
-        features_to_display = tuples[-top_k:][::-1] + tuples[:top_k][::-1]
+    def _neg_shap_value_magnitude(shap_tuple):
```

Comment:
Is this done strictly to sort the features w/ largest SHAP values first? If so, maybe we don't need this, and can use:
```
tuples = sorted(tuples, key=lambda x: x[0], reverse=True)
```

(Syntax not guaranteed ahaha)

Reply:
I originally went with a `def` instead of a `lambda` because I thought our linting prevented lambdas but it prevents assigning to a lambda, not using it as a function argument.

So this is not allowed:
```python
first_value_abs = lambda x: abs(x[0])
tuples = sorted(tuples, key=first_value, reverse=True)
```

But your suggestion is!

TIL something hehe. I'll update to use the in-line lambda.